pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -807,6 +808,7 @@
   ::  gate that produces the +per-server-event core from event information
   ::
   |=  [[our=@p eny=@ =duct now=@da scry=sley] state=server-state]
+  =/  eyre-id  (scot %ta (cat 3 'eyre_' (scot %uv (sham duct))))
```

Comment:
Noted elsewhere, but worth noting again: %eyre doesn't not currently ensure duct consistency for the lifetime of a channel. There's a good chance this will at least create duplicate subscriptions. If used as an id for an app with aa dojo-like interface, duct mismatches may also cause crashes.

Reply:
This is only for handle-http-request (ie lib/server-style) pokes, so apps don't have access to the duct/bone anyway.  Subscriptions to apps use a separate interface that's handled entirely by eyre and is indistinguishable to the app from a normal subscription.  I assume eyre ensures duct consistency for these.

For example, soto serves only static html/js/css/png via this mechanism.  To maintain a session with dojo, it generates a random id and passes that along with the subscription and pokes:

https://github.com/urbit/urbit/blob/e3005eaffa31f269467a8da31c31f972c5367f1f/pkg/interface/soto/src/index.js#L9
https://github.com/urbit/urbit/blob/e3005eaffa31f269467a8da31c31f972c5367f1f/pkg/interface/soto/src/js/subscription.js#L15
https://github.com/urbit/urbit/blob/e3005eaffa31f269467a8da31c31f972c5367f1f/pkg/interface/soto/src/js/subscription.js#L27
https://github.com/urbit/urbit/blob/e3005eaffa31f269467a8da31c31f972c5367f1f/pkg/interface/soto/src/js/api.js#L33-L35

@matildepark fyi